gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -861,6 +861,33 @@ MM_VerboseHandlerOutput::handleConcurrentStart(J9HookInterface** hook, UDATA eve
 	exitAtomicReportingBlock();
 }
 
+
+void
+MM_VerboseHandlerOutput::handleConcurrentEndInternal(J9HookInterface** hook, UDATA eventNum, void* eventData)
+{
+	MM_ConcurrentPhaseEndEvent *event = (MM_ConcurrentPhaseEndEvent *)eventData;
+	MM_ConcurrentPhaseStatsBase *stats = (MM_ConcurrentPhaseStatsBase *)event->concurrentStats;
```

Comment:
Sorry I missed these earlier.... Are these stats / event structs under the concurrent scavenger build define? Should they be?

Reply:
Implementation/body of handleConcurrentEndInternal (in base VGC handler) could indeed be under CONCURRENT_SCAVENGER compile flag - it's all CS specific. It does not have to be though, since MM_ConcurrentPhaseStatsBase (or anything else in the method) is not CS specific (the stats struct is also used by Balanced GC).

Should I add it?

Reply:
As long as nothing is CS specific and non CS specs will compile it is ok. I would rather not have things under if def if possible.